docs/source/examples/simple-landscape-braket.md

```diff
@@ -0,0 +1,203 @@
+
+This tutorial shows an example in which the energy landscape for a two-qubit variational circuit is explored with and without error mitigation, using Braket as our frontend.
+
+```{code-cell} ipython3
+import matplotlib.pyplot as plt
+import numpy as np
+
+from braket.devices import LocalSimulator
+from braket.circuits import Circuit, Noise, Observable
+
+from mitiq.zne import mitigate_executor
+from mitiq.zne.inference import RichardsonFactory
+```
+
+## Defining the ideal variational circuit in Qiskit
+
++++
+
+We define a function which returns a simple two-qubit variational circuit depending on a single parameter  $\gamma$ ("gamma").
+
+```{code-cell} ipython3
+def variational_circuit(gamma: float) -> Circuit:
+    """Returns a two-qubit circuit for a given variational parameter.
+
+    Args:
+        gamma: The variational parameter.
+
+    Returns:
+        The two-qubit circuit with a fixed gamma.
+    """
+    my_circuit = Circuit()
+    my_circuit.rx(0, gamma)
+    my_circuit.cnot(0, 1)
+    my_circuit.rx(1, gamma)
+    my_circuit.cnot(0, 1)
+    my_circuit.rx(0, gamma)
+    
+    return my_circuit
+```
+
+We can visualize the circuit for a particular $\gamma$ as follows.
+
+```{code-cell} ipython3
+circuit = variational_circuit(gamma=np.pi)
+print(circuit)
+```
+
+## Defining the executor functions with and without noise
+To use error mitigation methods in Mitiq, we define an executor function which computes the expectation value of a simple Hamiltonian $H=Z \otimes Z$, i.e., Pauli-$Z$ on each qubit. To compare to the noiseless result, we define both a noiseless and a noisy executor below.
+
+```{code-cell} ipython3
+# Observable to measure
+z = np.diag([1, -1])
+hamiltonian = np.kron(z, z)
+
+def noiseless_executor(circ: Circuit) -> float:
+    """Simulates the execution of a circuit without noise.
+
+    Args:
+        circ: The input circuit.
+
+    Returns:
+        The expectation value of the ZZ observable.
+    """
+    device = LocalSimulator('braket_dm')
+    # Evaluate the ZZ expectation value
+    circ.expectation(observable=Observable.Z() @ Observable.Z(), target=range(2))
+    
+    task = device.run(circ, shots = 0)
+    result = task.result()
+    return result.values
+
+# Strength of noise channel
+noise_level = 0.04
+
+def executor_with_noise(circ: Circuit) -> float:
+    """Simulates the execution of a circuit with depolarizing noise.
+
+    Args:
+        circ: The input circuit.
+
+    Returns:
+        The expectation value of the ZZ observable.
+    """
+    # Add depolarizing noise to the circuit
+    noise = Noise.Depolarizing(probability=noise_level)
+    circ.apply_gate_noise(noise)
+    # Use the noiseless_executor function to return the expectation value of the ZZ observable for the noisy circuit
+    return noiseless_executor(circ)
+```
+
+```{note}
+The above code block uses depolarizing noise, but any Braket `Noise` channel can be substituted in.
+```
```

Comment:
If you can find a link here to add which details further braket noise channels, that would be a great add.

Reply:
Sounds good. There's two options
- Official docs : [link](https://amazon-braket-sdk-python.readthedocs.io/en/latest/_apidoc/braket.circuits.noise.html)
- Notebook Example for Simulating Noise on Braket: [link](https://github.com/aws/amazon-braket-examples/blob/main/examples/braket_features/Simulating_Noise_On_Amazon_Braket.ipynb)

Which one do you prefer? The notebook example is probably more readable

Reply:
I agree the notebook is more readable, but I worry it's less permanent. Let's go with the official docs for now.